commons/src/main/java/gov/cms/qpp/conversion/model/validation/MeasureConfigs.java

```diff
@@ -2,26 +2,38 @@
 
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import gov.cms.qpp.conversion.util.EnvironmentHelper;
+import org.apache.lucene.search.spell.SpellChecker;
+import org.apache.lucene.store.FSDirectory;
 import org.reflections.util.ClasspathHelper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
 public class MeasureConfigs {
+	static final String MEASURES_INDEX_DIR = "MEASURES_INDEX_DIR";
```

Comment:
Can you add this to the `.env` file with a description?

Reply:
sure